packages/dart/lib/src/telemetry_processing/telemetry_buffer.dart

```diff
@@ -12,7 +13,7 @@ abstract class TelemetryBuffer<T> {
   void add(T item);
 
   /// When executed immediately sends all buffered items to Sentry and clears the buffer.
-  FutureOr<void> clear();
+  FutureOr<void> flush();
```

Comment:
@denrase I doubled checked cocoa and imo flush would fit better here. 

their architecture is a bit different so they call it clear because it actually only clears the buffer and not send it and their "flush" is done through the delegate

our 'flush' actually sends it and clears as well